server/src/monitoring/appdynamics.js

```diff
@@ -0,0 +1,61 @@
+;/* Version 6ed9e8be6594938a3179b249648f60c0 v:4.1.8.7, c:633a5cf8179243e14906a663132c69e8598f7c88, b:13635 n:52-4.1.8.next-controller */(function(){(function(){if(!window.ADRUM&&!0!==window["adrum-disable"]){var g=window.ADRUM={};window["adrum-start-time"]=window["adrum-start-time"]||(new Date).getTime();(function(a){(function(a){a.tc=function(){for(var a=[],c=0;c<arguments.length;c++)a[c-0]=arguments[c];for(c=0;c<a.length;c++){var b=a[c];b&&b.H()}}})(a.monitor||(a.monitor={}))})(g||(g={}));(function(a){a=a.conf||(a.conf={});a.beaconUrlHttp="http://col-eum.monitoring-appdynamics.zalan.do:80";a.beaconUrlHttps="https://col-eum.monitoring-appdynamics.zalan.do:443";a.corsEndpointPath="/eumcollector/beacons/browser/v1";
```

Comment:
These `appdynamics.zalan.do` URLs look like they should be set via a configuration option.